docs/application/dotnet/getting-started/runtime.md

```diff
@@ -0,0 +1,24 @@
+<a id="runtime"></a>
+# Managed Runtime Advantages
+
+A managed runtime offers the following advantages to your application development:
+
+- **Faster development**
+
+  Application development is accelerated since the managed runtime handles many functions that otherwise have to be coded into the application. The Tizen Native API and Tizen Web API already provide this benefit.
+
+- **Safer code**
+
+  Managed runtimes can handle, for example, bound checking, type safety, garbage collection, memory protection services, and objects being invoked directly. The Tizen Web API already provides some of these benefits, but not type safety, as JavaScript is not a strongly typed language.
+
+- **Lower deployment costs**
+
+  Component-based architecture makes it easier and faster to deploy applications in an enterprise environment characterized by multiple platforms, devices, and legacy systems.
+
+- **Better quality software**
+
+  Managed runtimes free you to focus on the business logic and code specific to the application, while reducing the number of coding errors.
```

Comment:
Suggestion: Managed runtime helps you to focus on the business logic and the code specific to the application, while reducing the number of errors in the code.

Reply:
Global comment:
Change  Managed runtimes to  Managed runtime.

Reply:
bound checking